module.exports = {
    DEFAULT_ERROR: 'En estos momentos existen dificultades, intente mas tarde'
}